{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "df69750c-0350-4c4a-a207-24317de635c3",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "c54504cf-b6b9-4023-a807-b0c27deb21b4",
   "metadata": {},
   "outputs": [],
   "source": [
    "data = pd.read_csv('AlphaMissense_hg38.tsv', sep='\\t', skiprows=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "ac7452d8-0bd6-4651-9837-f90d2e9fa9b4",
   "metadata": {},
   "outputs": [],
   "source": [
    "def pathogenicity_score(data, gene, variant):\n",
    "    reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
    "    score = np.mean(reduced_data['am_pathogenicity'])\n",
    "    return score"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "367b65fe-d973-42a4-a6f9-ae7aed225c2a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#CHROM</th>\n",
       "      <th>POS</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>genome</th>\n",
       "      <th>uniprot_id</th>\n",
       "      <th>transcript_id</th>\n",
       "      <th>protein_variant</th>\n",
       "      <th>am_pathogenicity</th>\n",
       "      <th>am_class</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2L</td>\n",
       "      <td>0.2937</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2L</td>\n",
       "      <td>0.2937</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2M</td>\n",
       "      <td>0.3296</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69095</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2A</td>\n",
       "      <td>0.2609</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69095</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2E</td>\n",
       "      <td>0.2922</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2013</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70004</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305C</td>\n",
       "      <td>0.0907</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2014</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70004</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305S</td>\n",
       "      <td>0.1053</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2015</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70004</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305Y</td>\n",
       "      <td>0.0795</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2016</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70005</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305L</td>\n",
       "      <td>0.3187</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2017</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70005</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305L</td>\n",
       "      <td>0.3187</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2018 rows × 10 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     #CHROM    POS REF ALT genome uniprot_id      transcript_id  \\\n",
       "0      chr1  69094   G   T   hg38     Q8NH21  ENST00000335137.4   \n",
       "1      chr1  69094   G   C   hg38     Q8NH21  ENST00000335137.4   \n",
       "2      chr1  69094   G   A   hg38     Q8NH21  ENST00000335137.4   \n",
       "3      chr1  69095   T   C   hg38     Q8NH21  ENST00000335137.4   \n",
       "4      chr1  69095   T   A   hg38     Q8NH21  ENST00000335137.4   \n",
       "...     ...    ...  ..  ..    ...        ...                ...   \n",
       "2013   chr1  70004   T   G   hg38     Q8NH21  ENST00000335137.4   \n",
       "2014   chr1  70004   T   C   hg38     Q8NH21  ENST00000335137.4   \n",
       "2015   chr1  70004   T   A   hg38     Q8NH21  ENST00000335137.4   \n",
       "2016   chr1  70005   T   G   hg38     Q8NH21  ENST00000335137.4   \n",
       "2017   chr1  70005   T   A   hg38     Q8NH21  ENST00000335137.4   \n",
       "\n",
       "     protein_variant  am_pathogenicity       am_class  \n",
       "0                V2L            0.2937  likely_benign  \n",
       "1                V2L            0.2937  likely_benign  \n",
       "2                V2M            0.3296  likely_benign  \n",
       "3                V2A            0.2609  likely_benign  \n",
       "4                V2E            0.2922  likely_benign  \n",
       "...              ...               ...            ...  \n",
       "2013           F305C            0.0907  likely_benign  \n",
       "2014           F305S            0.1053  likely_benign  \n",
       "2015           F305Y            0.0795  likely_benign  \n",
       "2016           F305L            0.3187  likely_benign  \n",
       "2017           F305L            0.3187  likely_benign  \n",
       "\n",
       "[2018 rows x 10 columns]"
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data[data['uniprot_id'] == 'Q8NH21']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "4d7f9787-8792-4340-9824-9e088b0c3477",
   "metadata": {},
   "outputs": [],
   "source": [
    "uniprot_unique = data['uniprot_id'].unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "4ee145c1-5f5a-49e2-a84a-38098924c79b",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Q8NH21\n",
      "Q96NU1\n",
      "Q9Y3T9\n",
      "Q6TDP4\n",
      "Q494U1\n",
      "Q5SV97\n",
      "Q9HCC6\n",
      "P05161\n",
      "E7ERA6\n",
      "Q96HA4\n",
      "Q6ZVT0\n",
      "Q9Y5U5\n",
      "P43489\n",
      "Q9BRK5\n",
      "Q96L58\n",
      "Q5T7M4\n",
      "Q8N2K1\n",
      "P51172\n",
      "Q96P50\n",
      "Q8N0Z8\n",
      "Q5TA45\n",
      "Q5TA50\n",
      "Q7RTX0\n",
      "O14640\n",
      "Q9BRK3\n",
      "Q9NWT8\n",
      "Q96S94\n",
      "Q9BYC9\n",
      "E5RJM6\n",
      "A6NKF7\n",
      "Q6PCB0\n",
      "Q5T2N8\n",
      "Q5T9A4\n",
      "Q9NVI7\n",
      "Q5SV17\n",
      "Q9NP77\n",
      "F2Z333\n",
      "O75900\n",
      "P21127\n",
      "P0CK96\n",
      "Q9UQ88\n",
      "P0CK97\n",
      "O95544\n",
      "P62873\n",
      "Q8TD86\n",
      "Q8NDY8\n",
      "Q9C0B2\n",
      "O14764\n",
      "Q05513\n",
      "Q6NZ36\n",
      "P12755\n",
      "Q5T089\n",
      "O15258\n",
      "O60683\n",
      "O75038\n",
      "Q9NVE7\n",
      "Q5TA89\n",
      "Q92956\n",
      "Q8TBF2\n",
      "Q495T6\n",
      "A8MYJ7\n",
      "Q8TDY3\n",
      "Q9HAZ2\n",
      "Q5VV41\n",
      "O75095\n",
      "Q5T0D9\n",
      "Q9P2S5\n",
      "O15350\n",
      "Q2M243\n",
      "B2RUZ4\n",
      "Q8N1G4\n",
      "O60308\n",
      "O76075\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[68], line 4\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m i \u001b[38;5;129;01min\u001b[39;00m uniprot_unique:\n\u001b[1;32m      3\u001b[0m     \u001b[38;5;28mprint\u001b[39m(i)\n\u001b[0;32m----> 4\u001b[0m     uniprot_dic[i] \u001b[38;5;241m=\u001b[39m data[\u001b[43mdata\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43muniprot_id\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[38;5;241;43m==\u001b[39;49m\u001b[43mi\u001b[49m]\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/ops/common.py:76\u001b[0m, in \u001b[0;36m_unpack_zerodim_and_defer.<locals>.new_method\u001b[0;34m(self, other)\u001b[0m\n\u001b[1;32m     72\u001b[0m             \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mNotImplemented\u001b[39m\n\u001b[1;32m     74\u001b[0m other \u001b[38;5;241m=\u001b[39m item_from_zerodim(other)\n\u001b[0;32m---> 76\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mmethod\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mother\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/arraylike.py:40\u001b[0m, in \u001b[0;36mOpsMixin.__eq__\u001b[0;34m(self, other)\u001b[0m\n\u001b[1;32m     38\u001b[0m \u001b[38;5;129m@unpack_zerodim_and_defer\u001b[39m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m__eq__\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m     39\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m__eq__\u001b[39m(\u001b[38;5;28mself\u001b[39m, other):\n\u001b[0;32m---> 40\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_cmp_method\u001b[49m\u001b[43m(\u001b[49m\u001b[43mother\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43moperator\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43meq\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/series.py:6119\u001b[0m, in \u001b[0;36mSeries._cmp_method\u001b[0;34m(self, other, op)\u001b[0m\n\u001b[1;32m   6116\u001b[0m lvalues \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_values\n\u001b[1;32m   6117\u001b[0m rvalues \u001b[38;5;241m=\u001b[39m extract_array(other, extract_numpy\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m, extract_range\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m)\n\u001b[0;32m-> 6119\u001b[0m res_values \u001b[38;5;241m=\u001b[39m \u001b[43mops\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcomparison_op\u001b[49m\u001b[43m(\u001b[49m\u001b[43mlvalues\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mrvalues\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mop\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   6121\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_construct_result(res_values, name\u001b[38;5;241m=\u001b[39mres_name)\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/ops/array_ops.py:344\u001b[0m, in \u001b[0;36mcomparison_op\u001b[0;34m(left, right, op)\u001b[0m\n\u001b[1;32m    341\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m invalid_comparison(lvalues, rvalues, op)\n\u001b[1;32m    343\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m lvalues\u001b[38;5;241m.\u001b[39mdtype \u001b[38;5;241m==\u001b[39m \u001b[38;5;28mobject\u001b[39m \u001b[38;5;129;01mor\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(rvalues, \u001b[38;5;28mstr\u001b[39m):\n\u001b[0;32m--> 344\u001b[0m     res_values \u001b[38;5;241m=\u001b[39m \u001b[43mcomp_method_OBJECT_ARRAY\u001b[49m\u001b[43m(\u001b[49m\u001b[43mop\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mlvalues\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mrvalues\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    346\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    347\u001b[0m     res_values \u001b[38;5;241m=\u001b[39m _na_arithmetic_op(lvalues, rvalues, op, is_cmp\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m)\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/ops/array_ops.py:129\u001b[0m, in \u001b[0;36mcomp_method_OBJECT_ARRAY\u001b[0;34m(op, x, y)\u001b[0m\n\u001b[1;32m    127\u001b[0m     result \u001b[38;5;241m=\u001b[39m libops\u001b[38;5;241m.\u001b[39mvec_compare(x\u001b[38;5;241m.\u001b[39mravel(), y\u001b[38;5;241m.\u001b[39mravel(), op)\n\u001b[1;32m    128\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m--> 129\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[43mlibops\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mscalar_compare\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mravel\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43my\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mop\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    130\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m result\u001b[38;5;241m.\u001b[39mreshape(x\u001b[38;5;241m.\u001b[39mshape)\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "uniprot_dic = {}\n",
    "for i in uniprot_unique:\n",
    "    print(i)\n",
    "    uniprot_dic[i] = data[data['uniprot_id']==i]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "id": "7713db86-cfbc-4647-a3af-93954a739e70",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#CHROM</th>\n",
       "      <th>POS</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>genome</th>\n",
       "      <th>uniprot_id</th>\n",
       "      <th>transcript_id</th>\n",
       "      <th>protein_variant</th>\n",
       "      <th>am_pathogenicity</th>\n",
       "      <th>am_class</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2L</td>\n",
       "      <td>0.2937</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2L</td>\n",
       "      <td>0.2937</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2M</td>\n",
       "      <td>0.3296</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69095</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2A</td>\n",
       "      <td>0.2609</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69095</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2E</td>\n",
       "      <td>0.2922</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2013</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70004</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305C</td>\n",
       "      <td>0.0907</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2014</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70004</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305S</td>\n",
       "      <td>0.1053</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2015</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70004</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305Y</td>\n",
       "      <td>0.0795</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2016</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70005</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305L</td>\n",
       "      <td>0.3187</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2017</th>\n",
       "      <td>chr1</td>\n",
       "      <td>70005</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>F305L</td>\n",
       "      <td>0.3187</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2018 rows × 10 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     #CHROM    POS REF ALT genome uniprot_id      transcript_id  \\\n",
       "0      chr1  69094   G   T   hg38     Q8NH21  ENST00000335137.4   \n",
       "1      chr1  69094   G   C   hg38     Q8NH21  ENST00000335137.4   \n",
       "2      chr1  69094   G   A   hg38     Q8NH21  ENST00000335137.4   \n",
       "3      chr1  69095   T   C   hg38     Q8NH21  ENST00000335137.4   \n",
       "4      chr1  69095   T   A   hg38     Q8NH21  ENST00000335137.4   \n",
       "...     ...    ...  ..  ..    ...        ...                ...   \n",
       "2013   chr1  70004   T   G   hg38     Q8NH21  ENST00000335137.4   \n",
       "2014   chr1  70004   T   C   hg38     Q8NH21  ENST00000335137.4   \n",
       "2015   chr1  70004   T   A   hg38     Q8NH21  ENST00000335137.4   \n",
       "2016   chr1  70005   T   G   hg38     Q8NH21  ENST00000335137.4   \n",
       "2017   chr1  70005   T   A   hg38     Q8NH21  ENST00000335137.4   \n",
       "\n",
       "     protein_variant  am_pathogenicity       am_class  \n",
       "0                V2L            0.2937  likely_benign  \n",
       "1                V2L            0.2937  likely_benign  \n",
       "2                V2M            0.3296  likely_benign  \n",
       "3                V2A            0.2609  likely_benign  \n",
       "4                V2E            0.2922  likely_benign  \n",
       "...              ...               ...            ...  \n",
       "2013           F305C            0.0907  likely_benign  \n",
       "2014           F305S            0.1053  likely_benign  \n",
       "2015           F305Y            0.0795  likely_benign  \n",
       "2016           F305L            0.3187  likely_benign  \n",
       "2017           F305L            0.3187  likely_benign  \n",
       "\n",
       "[2018 rows x 10 columns]"
      ]
     },
     "execution_count": 67,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data[data['uniprot_id']==uniprot_unique[0]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "eee0c68c-be37-46b1-9480-7c2332996355",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/2797515150.py:1: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  data1 = data[data['uniprot_id']=='Q8NH21'][data['protein_variant']=='V2L']\n"
     ]
    }
   ],
   "source": [
    "data1 = data[data['uniprot_id']=='Q8NH21'][data['protein_variant']=='V2L']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "85981c4d-097d-4346-aa47-a2cfffeeb9b1",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "np.float64(0.2937)"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pathogenicity_score(data, 'Q8NH21', 'V2L')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "8a16ecde-72ab-4221-a305-740c628a0dbf",
   "metadata": {},
   "outputs": [],
   "source": [
    "data_short = data.iloc[:][:1000]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "ec021221-6e54-49a7-8cbf-17d5367d184c",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#CHROM</th>\n",
       "      <th>POS</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>genome</th>\n",
       "      <th>uniprot_id</th>\n",
       "      <th>transcript_id</th>\n",
       "      <th>protein_variant</th>\n",
       "      <th>am_pathogenicity</th>\n",
       "      <th>am_class</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2L</td>\n",
       "      <td>0.2937</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2L</td>\n",
       "      <td>0.2937</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69094</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2M</td>\n",
       "      <td>0.3296</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69095</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2A</td>\n",
       "      <td>0.2609</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69095</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2E</td>\n",
       "      <td>0.2922</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69095</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>V2G</td>\n",
       "      <td>0.2030</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69097</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>T3A</td>\n",
       "      <td>0.0929</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69097</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>T3P</td>\n",
       "      <td>0.1264</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69097</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>T3S</td>\n",
       "      <td>0.0979</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>chr1</td>\n",
       "      <td>69098</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>hg38</td>\n",
       "      <td>Q8NH21</td>\n",
       "      <td>ENST00000335137.4</td>\n",
       "      <td>T3N</td>\n",
       "      <td>0.1121</td>\n",
       "      <td>likely_benign</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  #CHROM    POS REF ALT genome uniprot_id      transcript_id protein_variant  \\\n",
       "0   chr1  69094   G   T   hg38     Q8NH21  ENST00000335137.4             V2L   \n",
       "1   chr1  69094   G   C   hg38     Q8NH21  ENST00000335137.4             V2L   \n",
       "2   chr1  69094   G   A   hg38     Q8NH21  ENST00000335137.4             V2M   \n",
       "3   chr1  69095   T   C   hg38     Q8NH21  ENST00000335137.4             V2A   \n",
       "4   chr1  69095   T   A   hg38     Q8NH21  ENST00000335137.4             V2E   \n",
       "5   chr1  69095   T   G   hg38     Q8NH21  ENST00000335137.4             V2G   \n",
       "6   chr1  69097   A   G   hg38     Q8NH21  ENST00000335137.4             T3A   \n",
       "7   chr1  69097   A   C   hg38     Q8NH21  ENST00000335137.4             T3P   \n",
       "8   chr1  69097   A   T   hg38     Q8NH21  ENST00000335137.4             T3S   \n",
       "9   chr1  69098   C   A   hg38     Q8NH21  ENST00000335137.4             T3N   \n",
       "\n",
       "   am_pathogenicity       am_class  \n",
       "0            0.2937  likely_benign  \n",
       "1            0.2937  likely_benign  \n",
       "2            0.3296  likely_benign  \n",
       "3            0.2609  likely_benign  \n",
       "4            0.2922  likely_benign  \n",
       "5            0.2030  likely_benign  \n",
       "6            0.0929  likely_benign  \n",
       "7            0.1264  likely_benign  \n",
       "8            0.0979  likely_benign  \n",
       "9            0.1121  likely_benign  "
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_short"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "d9736ce8-6c50-4f53-86b2-2a7aad46fadb",
   "metadata": {},
   "outputs": [],
   "source": [
    "patient = pd.read_csv('alpha_missense_input.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "id": "c99ce195-66a8-4259-8a1e-b9bdba470b7e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "9593"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(patient['Patient ID'].unique())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "id": "6ca1e75b-1434-4286-94ef-50f2f2adbbd7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Patient ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Protein Change</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>5870</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>P00519</td>\n",
       "      <td>E581Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5871</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>P29320</td>\n",
       "      <td>D838N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5872</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>P54756</td>\n",
       "      <td>D136Y</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5873</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>P54762</td>\n",
       "      <td>V916L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5874</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>Q12809</td>\n",
       "      <td>Q286E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5875</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>Q14517</td>\n",
       "      <td>Q16*</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5876</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>Q12879</td>\n",
       "      <td>A638D</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5877</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>P35968</td>\n",
       "      <td>L1194Afs*15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5878</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>O14746</td>\n",
       "      <td>P320L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5879</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>P04637</td>\n",
       "      <td>F113C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5880</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>O14686</td>\n",
       "      <td>P2168S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5881</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>O95835</td>\n",
       "      <td>P445L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5882</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>Q93074</td>\n",
       "      <td>A1090V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5883</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>Q6N021</td>\n",
       "      <td>E524*</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5884</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>Q8NFU7</td>\n",
       "      <td>Q1564E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5885</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>Q8TEK3</td>\n",
       "      <td>H703L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5886</th>\n",
       "      <td>P-0000315</td>\n",
       "      <td>Q68CP9</td>\n",
       "      <td>R1677L</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     Patient ID    Gene Protein Change\n",
       "5870  P-0000315  P00519          E581Q\n",
       "5871  P-0000315  P29320          D838N\n",
       "5872  P-0000315  P54756          D136Y\n",
       "5873  P-0000315  P54762          V916L\n",
       "5874  P-0000315  Q12809          Q286E\n",
       "5875  P-0000315  Q14517           Q16*\n",
       "5876  P-0000315  Q12879          A638D\n",
       "5877  P-0000315  P35968    L1194Afs*15\n",
       "5878  P-0000315  O14746          P320L\n",
       "5879  P-0000315  P04637          F113C\n",
       "5880  P-0000315  O14686         P2168S\n",
       "5881  P-0000315  O95835          P445L\n",
       "5882  P-0000315  Q93074         A1090V\n",
       "5883  P-0000315  Q6N021          E524*\n",
       "5884  P-0000315  Q8NFU7         Q1564E\n",
       "5885  P-0000315  Q8TEK3          H703L\n",
       "5886  P-0000315  Q68CP9         R1677L"
      ]
     },
     "execution_count": 74,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "patient['Patient ID'].unique()[100]\n",
    "patient[patient['Patient ID']=='P-0000315']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "id": "250ff81c-4f11-4ecb-9408-3a38b91fd7f9",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "20\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "30\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "40\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "60\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "70\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "80\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "90\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "100\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "110\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "120\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "130\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "140\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "150\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "160\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "170\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "180\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "190\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "210\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "220\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "230\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "240\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "250\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "260\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
      "/tmp/ipykernel_1346579/1221900531.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.\n",
      "  reduced_data = data[data['uniprot_id']==gene][data['protein_variant']==variant]\n",
      "/tmp/ipykernel_1346579/3088895220.py:4: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`\n",
      "  gene_score.append(pathogenicity_score(data, i[1], i[2]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "270\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[75], line 4\u001b[0m\n\u001b[1;32m      2\u001b[0m count \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m0\u001b[39m\n\u001b[1;32m      3\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m _, i \u001b[38;5;129;01min\u001b[39;00m patient\u001b[38;5;241m.\u001b[39miterrows():\n\u001b[0;32m----> 4\u001b[0m     gene_score\u001b[38;5;241m.\u001b[39mappend(\u001b[43mpathogenicity_score\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdata\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mi\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m1\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mi\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m2\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m)\u001b[49m)\n\u001b[1;32m      5\u001b[0m     count\u001b[38;5;241m+\u001b[39m\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m1\u001b[39m\n\u001b[1;32m      6\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m count \u001b[38;5;241m%\u001b[39m \u001b[38;5;241m10\u001b[39m \u001b[38;5;241m==\u001b[39m \u001b[38;5;241m0\u001b[39m:\n",
      "Cell \u001b[0;32mIn[32], line 2\u001b[0m, in \u001b[0;36mpathogenicity_score\u001b[0;34m(data, gene, variant)\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mpathogenicity_score\u001b[39m(data, gene, variant):\n\u001b[0;32m----> 2\u001b[0m     reduced_data \u001b[38;5;241m=\u001b[39m data[\u001b[43mdata\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43muniprot_id\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[38;5;241;43m==\u001b[39;49m\u001b[43mgene\u001b[49m][data[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mprotein_variant\u001b[39m\u001b[38;5;124m'\u001b[39m]\u001b[38;5;241m==\u001b[39mvariant]\n\u001b[1;32m      3\u001b[0m     score \u001b[38;5;241m=\u001b[39m np\u001b[38;5;241m.\u001b[39mmean(reduced_data[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mam_pathogenicity\u001b[39m\u001b[38;5;124m'\u001b[39m])\n\u001b[1;32m      4\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m score\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/ops/common.py:76\u001b[0m, in \u001b[0;36m_unpack_zerodim_and_defer.<locals>.new_method\u001b[0;34m(self, other)\u001b[0m\n\u001b[1;32m     72\u001b[0m             \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mNotImplemented\u001b[39m\n\u001b[1;32m     74\u001b[0m other \u001b[38;5;241m=\u001b[39m item_from_zerodim(other)\n\u001b[0;32m---> 76\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mmethod\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mother\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/arraylike.py:40\u001b[0m, in \u001b[0;36mOpsMixin.__eq__\u001b[0;34m(self, other)\u001b[0m\n\u001b[1;32m     38\u001b[0m \u001b[38;5;129m@unpack_zerodim_and_defer\u001b[39m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m__eq__\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m     39\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m__eq__\u001b[39m(\u001b[38;5;28mself\u001b[39m, other):\n\u001b[0;32m---> 40\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_cmp_method\u001b[49m\u001b[43m(\u001b[49m\u001b[43mother\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43moperator\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43meq\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/series.py:6119\u001b[0m, in \u001b[0;36mSeries._cmp_method\u001b[0;34m(self, other, op)\u001b[0m\n\u001b[1;32m   6116\u001b[0m lvalues \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_values\n\u001b[1;32m   6117\u001b[0m rvalues \u001b[38;5;241m=\u001b[39m extract_array(other, extract_numpy\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m, extract_range\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m)\n\u001b[0;32m-> 6119\u001b[0m res_values \u001b[38;5;241m=\u001b[39m \u001b[43mops\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcomparison_op\u001b[49m\u001b[43m(\u001b[49m\u001b[43mlvalues\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mrvalues\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mop\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   6121\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_construct_result(res_values, name\u001b[38;5;241m=\u001b[39mres_name)\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/ops/array_ops.py:344\u001b[0m, in \u001b[0;36mcomparison_op\u001b[0;34m(left, right, op)\u001b[0m\n\u001b[1;32m    341\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m invalid_comparison(lvalues, rvalues, op)\n\u001b[1;32m    343\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m lvalues\u001b[38;5;241m.\u001b[39mdtype \u001b[38;5;241m==\u001b[39m \u001b[38;5;28mobject\u001b[39m \u001b[38;5;129;01mor\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(rvalues, \u001b[38;5;28mstr\u001b[39m):\n\u001b[0;32m--> 344\u001b[0m     res_values \u001b[38;5;241m=\u001b[39m \u001b[43mcomp_method_OBJECT_ARRAY\u001b[49m\u001b[43m(\u001b[49m\u001b[43mop\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mlvalues\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mrvalues\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    346\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    347\u001b[0m     res_values \u001b[38;5;241m=\u001b[39m _na_arithmetic_op(lvalues, rvalues, op, is_cmp\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m)\n",
      "File \u001b[0;32m~/miniconda3/envs/hacklowe/lib/python3.9/site-packages/pandas/core/ops/array_ops.py:129\u001b[0m, in \u001b[0;36mcomp_method_OBJECT_ARRAY\u001b[0;34m(op, x, y)\u001b[0m\n\u001b[1;32m    127\u001b[0m     result \u001b[38;5;241m=\u001b[39m libops\u001b[38;5;241m.\u001b[39mvec_compare(x\u001b[38;5;241m.\u001b[39mravel(), y\u001b[38;5;241m.\u001b[39mravel(), op)\n\u001b[1;32m    128\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m--> 129\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[43mlibops\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mscalar_compare\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mravel\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43my\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mop\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    130\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m result\u001b[38;5;241m.\u001b[39mreshape(x\u001b[38;5;241m.\u001b[39mshape)\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "gene_score = []\n",
    "count = 0\n",
    "for _, i in patient.iterrows():\n",
    "    gene_score.append(pathogenicity_score(data, i[1], i[2]))\n",
    "    count+=1\n",
    "    if count % 10 == 0:\n",
    "        print(count)\n",
    "    if count == 5886:\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "id": "c855ae7c-8458-44e4-91e4-ae06289dfb5b",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[nan,\n",
       " np.float64(0.6048),\n",
       " nan,\n",
       " np.float64(0.9979),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.0807),\n",
       " np.float64(0.1348),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.9826),\n",
       " np.float64(0.7967999999999998),\n",
       " np.float64(0.997),\n",
       " np.float64(0.6837),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.1637),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.6652),\n",
       " np.float64(0.2176),\n",
       " np.float64(0.098),\n",
       " nan,\n",
       " np.float64(0.3187),\n",
       " np.float64(0.9772),\n",
       " nan,\n",
       " np.float64(0.0637),\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.8563),\n",
       " np.float64(0.1016),\n",
       " np.float64(0.9525),\n",
       " np.float64(0.8879),\n",
       " np.float64(0.6124),\n",
       " np.float64(0.9612),\n",
       " np.float64(0.9815),\n",
       " np.float64(0.971),\n",
       " np.float64(0.4719),\n",
       " np.float64(0.1076),\n",
       " np.float64(0.1389),\n",
       " np.float64(0.8989),\n",
       " nan,\n",
       " np.float64(0.1852),\n",
       " np.float64(0.158),\n",
       " nan,\n",
       " np.float64(0.986),\n",
       " np.float64(0.0683),\n",
       " np.float64(0.1776),\n",
       " np.float64(0.0925),\n",
       " np.float64(0.998),\n",
       " np.float64(0.5769),\n",
       " nan,\n",
       " np.float64(0.0682),\n",
       " nan,\n",
       " np.float64(0.9122),\n",
       " np.float64(0.2373),\n",
       " np.float64(0.0514),\n",
       " nan,\n",
       " np.float64(0.0738),\n",
       " nan,\n",
       " np.float64(0.5884),\n",
       " np.float64(0.9998),\n",
       " np.float64(0.9994),\n",
       " np.float64(0.9974),\n",
       " np.float64(0.315),\n",
       " np.float64(0.9908),\n",
       " np.float64(0.974),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.0971),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.1366),\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.1936),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.9951),\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.1813),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.077),\n",
       " nan,\n",
       " np.float64(0.786),\n",
       " np.float64(0.3221),\n",
       " nan,\n",
       " np.float64(0.0653),\n",
       " nan,\n",
       " np.float64(0.0871),\n",
       " np.float64(0.1109),\n",
       " np.float64(0.0893),\n",
       " nan,\n",
       " np.float64(0.0866),\n",
       " nan,\n",
       " np.float64(0.9998),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.975),\n",
       " np.float64(0.8503),\n",
       " np.float64(0.5507),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.6815),\n",
       " np.float64(0.95),\n",
       " nan,\n",
       " np.float64(0.0959),\n",
       " np.float64(0.4111),\n",
       " np.float64(0.2323),\n",
       " np.float64(0.222),\n",
       " np.float64(0.9992),\n",
       " np.float64(0.8339),\n",
       " nan,\n",
       " np.float64(0.2875),\n",
       " np.float64(0.087),\n",
       " np.float64(0.0634),\n",
       " np.float64(0.3559),\n",
       " np.float64(0.0994),\n",
       " np.float64(0.0455),\n",
       " np.float64(0.9722),\n",
       " np.float64(0.0749),\n",
       " np.float64(0.1971),\n",
       " np.float64(0.6786),\n",
       " np.float64(0.1168),\n",
       " np.float64(0.4848),\n",
       " np.float64(0.0972),\n",
       " nan,\n",
       " np.float64(0.145),\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.7567),\n",
       " np.float64(0.0684),\n",
       " np.float64(0.1778),\n",
       " np.float64(0.9832),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.2193),\n",
       " np.float64(0.7935),\n",
       " np.float64(0.2674),\n",
       " np.float64(0.1472),\n",
       " np.float64(0.0823),\n",
       " np.float64(0.1087),\n",
       " np.float64(0.0791),\n",
       " np.float64(0.0949),\n",
       " np.float64(0.0851),\n",
       " np.float64(0.0783),\n",
       " np.float64(0.4017),\n",
       " np.float64(0.367),\n",
       " np.float64(0.8548),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.9993),\n",
       " np.float64(0.8634),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.9922),\n",
       " np.float64(0.9862),\n",
       " np.float64(0.342),\n",
       " np.float64(0.1982),\n",
       " np.float64(0.0937),\n",
       " np.float64(0.71),\n",
       " np.float64(0.881),\n",
       " np.float64(0.0888),\n",
       " np.float64(0.4547),\n",
       " nan,\n",
       " np.float64(0.0582),\n",
       " np.float64(0.0922),\n",
       " np.float64(0.0852),\n",
       " np.float64(0.937),\n",
       " np.float64(0.1667),\n",
       " np.float64(0.3382),\n",
       " np.float64(0.11),\n",
       " np.float64(0.9999),\n",
       " np.float64(0.879),\n",
       " np.float64(0.4714),\n",
       " nan,\n",
       " np.float64(0.8982),\n",
       " np.float64(0.4538),\n",
       " np.float64(0.964),\n",
       " np.float64(0.1189),\n",
       " np.float64(0.9898),\n",
       " np.float64(0.1148),\n",
       " np.float64(0.3222),\n",
       " np.float64(0.1016),\n",
       " np.float64(0.3954),\n",
       " np.float64(0.9899),\n",
       " np.float64(0.0963),\n",
       " np.float64(0.1641),\n",
       " nan,\n",
       " np.float64(0.0999),\n",
       " np.float64(0.1337),\n",
       " np.float64(0.084),\n",
       " np.float64(0.082),\n",
       " nan,\n",
       " np.float64(0.0759),\n",
       " np.float64(0.9513),\n",
       " np.float64(0.2551),\n",
       " np.float64(0.0673),\n",
       " np.float64(0.1139),\n",
       " np.float64(0.104),\n",
       " np.float64(0.998),\n",
       " nan,\n",
       " np.float64(0.3644),\n",
       " nan,\n",
       " np.float64(0.6449),\n",
       " nan,\n",
       " np.float64(0.1769),\n",
       " nan,\n",
       " np.float64(0.1563),\n",
       " np.float64(0.9911),\n",
       " np.float64(0.0977),\n",
       " np.float64(0.1549),\n",
       " np.float64(0.1035),\n",
       " np.float64(0.8752),\n",
       " np.float64(0.1923),\n",
       " np.float64(0.9977),\n",
       " np.float64(0.3639),\n",
       " np.float64(0.96),\n",
       " np.float64(0.695),\n",
       " np.float64(0.9844),\n",
       " np.float64(0.0728),\n",
       " np.float64(0.9924),\n",
       " np.float64(0.2548),\n",
       " np.float64(0.1872),\n",
       " np.float64(0.9565),\n",
       " np.float64(0.9995),\n",
       " np.float64(0.2034),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.9979),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.7925),\n",
       " np.float64(0.7729),\n",
       " np.float64(0.7514),\n",
       " np.float64(0.0683),\n",
       " np.float64(0.9161),\n",
       " np.float64(0.9171),\n",
       " np.float64(0.1659),\n",
       " np.float64(0.1026),\n",
       " np.float64(0.0959),\n",
       " np.float64(0.9615),\n",
       " np.float64(0.9876),\n",
       " np.float64(0.3838),\n",
       " np.float64(0.8895),\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.2108),\n",
       " np.float64(0.1362),\n",
       " nan,\n",
       " nan,\n",
       " nan,\n",
       " np.float64(0.1075),\n",
       " np.float64(0.0876),\n",
       " np.float64(0.0786)]"
      ]
     },
     "execution_count": 76,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_score"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "id": "69bb5d5d-1568-40c0-a6fe-f5d83c9f3bd1",
   "metadata": {},
   "outputs": [],
   "source": [
    "aditional = pd.DataFrame({\n",
    "    'Pathogenicity_Score':gene_score\n",
    "})\n",
    "new = pd.concat([patient, aditional], axis=1) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "717147ef-75e0-4710-ab98-f1739d397bfb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Patient ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Protein Change</th>\n",
       "      <th>Pathogenicity_Score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>P-0000507</td>\n",
       "      <td>P00519</td>\n",
       "      <td>K947Sfs*122</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>P-0000507</td>\n",
       "      <td>P04198</td>\n",
       "      <td>K446E</td>\n",
       "      <td>0.6048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>P-0000507</td>\n",
       "      <td>P27986</td>\n",
       "      <td>H385Ifs*11</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>P-0000507</td>\n",
       "      <td>P20936</td>\n",
       "      <td>L1009R</td>\n",
       "      <td>0.9979</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>P-0000507</td>\n",
       "      <td>O15550</td>\n",
       "      <td>F289Sfs*36</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78137</th>\n",
       "      <td>P-0001636</td>\n",
       "      <td>Q8NFG4</td>\n",
       "      <td>A445T</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78138</th>\n",
       "      <td>P-0001314</td>\n",
       "      <td>Q6R327</td>\n",
       "      <td>H1109Tfs*3</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78139</th>\n",
       "      <td>P-0001314</td>\n",
       "      <td>Q6R327</td>\n",
       "      <td>R485Q</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78140</th>\n",
       "      <td>P-0003138</td>\n",
       "      <td>Q6R327</td>\n",
       "      <td>*1709Lext*29</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78141</th>\n",
       "      <td>P-0005632</td>\n",
       "      <td>Q6R327</td>\n",
       "      <td>L847F</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>78142 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      Patient ID    Gene Protein Change  Pathogenicity_Score\n",
       "0      P-0000507  P00519    K947Sfs*122                  NaN\n",
       "1      P-0000507  P04198          K446E               0.6048\n",
       "2      P-0000507  P27986     H385Ifs*11                  NaN\n",
       "3      P-0000507  P20936         L1009R               0.9979\n",
       "4      P-0000507  O15550     F289Sfs*36                  NaN\n",
       "...          ...     ...            ...                  ...\n",
       "78137  P-0001636  Q8NFG4          A445T                  NaN\n",
       "78138  P-0001314  Q6R327     H1109Tfs*3                  NaN\n",
       "78139  P-0001314  Q6R327          R485Q                  NaN\n",
       "78140  P-0003138  Q6R327   *1709Lext*29                  NaN\n",
       "78141  P-0005632  Q6R327          L847F                  NaN\n",
       "\n",
       "[78142 rows x 4 columns]"
      ]
     },
     "execution_count": 79,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "new['Pathogenicity_Score'].isn"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "id": "fd7f4712-0e0f-4454-a2c0-e2be824a4e88",
   "metadata": {},
   "outputs": [],
   "source": [
    "oncogenic = pd.read_csv('oncogenic.csv', index_col=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "id": "9402189f-88b3-497b-b648-c065532c7b2a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Hugo Symbol</th>\n",
       "      <th>Oncogene</th>\n",
       "      <th>Tumor Suppressor Gene</th>\n",
       "      <th>UniProt</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ndex</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ABL1</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>P00519</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>MYCN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>P04198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>PIK3R1</td>\n",
       "      <td>No</td>\n",
       "      <td>Yes</td>\n",
       "      <td>P27986</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>557</th>\n",
       "      <td>RASA1</td>\n",
       "      <td>No</td>\n",
       "      <td>Yes</td>\n",
       "      <td>P20936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>KDM6A</td>\n",
       "      <td>No</td>\n",
       "      <td>Yes</td>\n",
       "      <td>O15550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>MAP3K14</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Q99558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>TMEM127</td>\n",
       "      <td>No</td>\n",
       "      <td>Yes</td>\n",
       "      <td>O75204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>543</th>\n",
       "      <td>PMAIP1</td>\n",
       "      <td>No</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Q13794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>H3C13</td>\n",
       "      <td>No</td>\n",
       "      <td>No</td>\n",
       "      <td>Q71DI3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>H3C1</td>\n",
       "      <td>No</td>\n",
       "      <td>No</td>\n",
       "      <td>P68431</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>410 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Hugo Symbol Oncogene Tumor Suppressor Gene UniProt\n",
       "ndex                                                   \n",
       "0           ABL1      Yes                    No  P00519\n",
       "166         MYCN      Yes                    No  P04198\n",
       "74        PIK3R1       No                   Yes  P27986\n",
       "557        RASA1       No                   Yes  P20936\n",
       "49         KDM6A       No                   Yes  O15550\n",
       "...          ...      ...                   ...     ...\n",
       "330      MAP3K14      Yes                    No  Q99558\n",
       "367      TMEM127       No                   Yes  O75204\n",
       "543       PMAIP1       No                   Yes  Q13794\n",
       "466        H3C13       No                    No  Q71DI3\n",
       "462         H3C1       No                    No  P68431\n",
       "\n",
       "[410 rows x 4 columns]"
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncogenic"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "71976312-f0c0-49e9-bc47-08b2f6421054",
   "metadata": {},
   "source": [
    "# Miscellaneous"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "f785ac03-7386-4ae9-b784-5ed78adfaa79",
   "metadata": {},
   "outputs": [],
   "source": [
    "tree = {}\n",
    "\n",
    "for iter, row in data_short.iterrows():\n",
    "    uniprot = row['uniprot_id']\n",
    "    variant = row['protein_variant']\n",
    "    pathogenicity = row['am_pathogenicity']\n",
    "    pclass = row['am_class']\n",
    "\n",
    "    if uniprot not in tree:\n",
    "        tree[uniprot] = {}\n",
    "    if variant not in tree[uniprot]:\n",
    "        tree[uniprot][variant] = []\n",
    "    \n",
    "    tree[uniprot][variant].append({'Pathogenicity_Score': pathogenicity, 'Class': pclass})\n",
    "\n",
    "if iter % 10 == 0:\n",
    "    print(iter)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "ebde3bcd-fc88-4eca-ab1e-5834e9c014bf",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "    \"Q8NH21\": {\n",
      "        \"V2L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2937,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2937,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V2M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3296,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V2A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2609,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V2E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2922,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V2G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.203,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T3A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0929,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T3P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1264,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T3S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0979,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0979,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T3N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1121,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T3I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2848,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E4Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1538,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E4K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1683,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E4A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2309,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E4G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2101,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E4V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3318,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E4D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2349,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2349,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F5I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2985,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F5L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.911,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.911,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.911,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F5V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4055,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F5C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6917,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F5S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7921,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F5Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3177,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I6L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1059,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I6F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1518,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I6V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0849,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I6N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2261,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I6S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2553,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I6T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2555,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I6M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1827,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F7I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1679,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F7L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3912,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3912,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3912,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F7V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2894,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F7C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5156,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F7S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.659,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F7Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3241,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L8M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1532,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L8V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1125,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L8R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0862,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L8Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.102,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L8P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3214,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G9R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5441,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G9C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4919,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G9S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.258,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G9A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3359,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G9D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4311,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G9V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4323,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L10I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.12,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L10F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1122,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L10V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1607,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L10R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4162,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L10H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4721,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L10P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3723,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S11A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.109,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S11P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1936,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S11T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.082,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S11C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.116,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S11F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2599,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S11Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1985,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D12N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0762,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D12H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1106,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D12Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1088,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D12A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1013,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D12G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.089,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D12V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1211,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D12E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.137,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.137,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S13A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1048,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S13P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1884,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S13T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0934,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S13C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1209,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S13F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1638,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S13Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1226,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q14E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0762,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q14K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0629,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q14R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0539,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q14L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0934,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q14P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0629,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q14H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1307,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1307,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E15Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0917,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E15K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0936,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E15A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1011,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E15G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0854,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E15V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1017,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E15D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1061,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1061,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L16I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0918,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L16F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.118,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L16V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1025,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L16R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1551,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L16H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2067,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L16P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2639,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q17E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0914,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q17K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1119,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q17R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1064,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q17L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2544,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q17P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.224,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q17H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.353,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.353,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"T18A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0721,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T18P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0667,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T18S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.093,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.093,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T18N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0809,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T18I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0623,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F19I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0722,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F19L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3601,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3601,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3601,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F19V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0742,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F19C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1338,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F19S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1386,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F19Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1532,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L20I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1452,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L20V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1542,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L20R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2927,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L20Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2705,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L20P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2908,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F21I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1968,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F21L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5486,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5486,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5486,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F21V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1745,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F21C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1825,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F21S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2087,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F21Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2166,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M22L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0695,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0695,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M22V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0518,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M22R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.323,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M22K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3524,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M22T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0885,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M22I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1243,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1243,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1243,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L23M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1202,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L23V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0765,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L23S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2118,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L23W\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2363,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L23F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0925,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0925,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F24I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1781,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F24L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6714,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6714,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6714,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F24V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.175,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F24C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.211,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F24S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2412,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F24Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2613,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F25I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0744,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F25L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.215,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.215,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.215,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F25V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0653,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F25C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0774,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F25S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0765,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F25Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2058,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V26I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0742,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V26L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0849,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0849,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V26A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1124,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V26E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6213,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"V26G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1313,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F27I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0932,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F27L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3886,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3886,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3886,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F27V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0996,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F27C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1167,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F27S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1762,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F27Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3018,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y28N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4298,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y28D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6077,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y28H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3718,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y28C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1703,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y28F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.135,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y28S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.304,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G29R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4756,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4756,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G29A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0587,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G29E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4103,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G29V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0645,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G30R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6014,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6014,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"G30A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0761,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G30E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4393,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G30V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1109,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I31L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2607,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I31F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3693,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I31V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1192,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I31N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2908,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I31S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2327,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I31T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1206,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I31M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.144,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V32L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1281,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1281,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V32M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1412,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V32A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1992,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V32E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6174,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"V32G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2395,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F33I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0809,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F33L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2311,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2311,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2311,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F33V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0799,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F33C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0992,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F33S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1011,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F33Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2486,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G34R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5448,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5448,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G34A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3303,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G34E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.536,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G34V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.528,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"N35D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6076,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"N35H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4345,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"N35Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5019,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"N35I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6153,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"N35S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3501,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"N35T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6255,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"N35K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.733,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.733,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"L36I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0989,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L36F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.101,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L36V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.108,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L36R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5715,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"L36H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4818,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L36P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.678,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"L37I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1286,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L37F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.121,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L37V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1262,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L37R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3536,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L37H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3619,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L37P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4021,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I38L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1886,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I38F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3696,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I38V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1255,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I38N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5139,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I38S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4291,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I38T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1743,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I38M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1946,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V39I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0809,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V39L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1573,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V39F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1488,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V39A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2452,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V39D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.634,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"V39G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2509,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I40L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0734,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0734,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I40V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0789,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I40R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2451,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I40K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3538,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I40T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1271,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I40M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.076,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T41A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0858,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T41P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2014,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T41S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1938,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T41R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3608,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"T41I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.215,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T41K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4653,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V42L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4212,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4212,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V42M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2404,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V42A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2473,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V42E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.598,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"V42G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3688,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V43I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0757,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V43L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1005,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1005,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V43A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.078,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V43E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2816,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V43G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.107,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S44A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0759,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S44P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3163,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S44T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0745,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S44C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0796,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S44F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1151,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S44Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1073,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D45N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1089,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D45H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2223,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D45Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2068,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D45A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2219,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D45G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2365,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D45V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2448,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D45E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2958,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2958,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S46A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0782,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S46P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0549,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S46T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0822,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S46C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0789,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S46F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1406,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S46Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1092,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H47N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1229,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H47D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2026,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H47Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1024,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H47R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0732,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H47L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1376,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H47P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1758,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H47Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1578,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1578,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L48I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2489,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L48F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2865,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L48V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2818,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L48R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3248,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L48H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4457,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L48P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4541,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H49N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2188,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H49D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3552,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H49Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.15,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H49R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.107,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H49L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.206,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H49P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.24,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H49Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2381,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2381,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S50A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1236,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S50P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3852,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S50T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0916,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S50C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1222,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S50F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2226,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S50Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1721,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"P51A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2188,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"P51S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2991,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"P51T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.291,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"P51R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.34,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"P51H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4088,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P51L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3179,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M52L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1883,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1883,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M52V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1163,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M52R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2054,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M52K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2378,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M52T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2776,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M52I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6207,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6207,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6207,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y53N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3451,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y53D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5506,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y53H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5509,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y53C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4464,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y53F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2045,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y53S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4909,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F54I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1441,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F54L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6702,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6702,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6702,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F54V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1591,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F54C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3074,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F54S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3739,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F54Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1731,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L55M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2211,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L55V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2714,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L55R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4315,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L55Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3857,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L55P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4179,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L56I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2692,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L56V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2833,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L56R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4701,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L56Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5497,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L56P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3905,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"A57P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5405,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"A57S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1386,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A57T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1388,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A57D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6084,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"A57G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1005,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A57V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2205,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N58D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3703,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"N58H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1523,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N58Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2715,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N58I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3613,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"N58S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1031,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N58T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2727,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N58K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6124,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6124,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"L59I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2469,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L59F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2887,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L59V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2703,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L59R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.388,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L59H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.481,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L59P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5246,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S60A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1165,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S60P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3948,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S60T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3883,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S60L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3469,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L61I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0939,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L61F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0735,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L61V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0857,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L61R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5285,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L61H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4859,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L61P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6409,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"I62L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0971,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I62F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1763,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I62V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1018,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I62N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2156,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I62S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1785,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I62T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.081,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I62M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0884,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D63N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5106,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D63H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6185,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"D63Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4398,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D63A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7096,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"D63G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5867,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"D63V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6485,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"D63E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6958,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6958,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"L64M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1202,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L64V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0997,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L64R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5959,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"L64Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.48,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L64P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6559,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"S65A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0787,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S65P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3591,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S65T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0919,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S65C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0429,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S65F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1111,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S65Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2248,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L66M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1896,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L66V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1286,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L66R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4289,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L66Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3274,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L66P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7089,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"S67A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0794,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S67P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2835,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S67T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1903,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S67C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1218,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S67F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5387,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S67Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5156,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S68A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1247,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S68P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5068,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S68T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.083,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S68L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1705,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V69I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V69L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3057,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V69F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2005,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V69A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1941,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V69D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.522,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V69G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2818,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T70A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1001,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T70P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1676,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T70S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.205,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T70R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.389,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"T70I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2252,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T70K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5195,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"A71P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3756,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"A71S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1241,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A71T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.082,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A71D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6126,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"A71G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1796,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A71V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0982,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"P72A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2857,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"P72S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4117,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P72T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3055,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"P72R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4184,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P72H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5008,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P72L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.273,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K73Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2044,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K73E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3361,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K73R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0814,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K73M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.388,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"K73T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3579,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"K73N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6429,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6429,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"M74L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1856,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1856,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M74V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2026,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M74R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5657,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"M74K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6068,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"M74T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5148,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M74I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6113,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6113,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6113,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"I75L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1082,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I75F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2346,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I75V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1398,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I75N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4216,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I75S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4921,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I75T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4167,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I75M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1056,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T76A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0648,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T76P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1794,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T76S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0956,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0956,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T76N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1274,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T76I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.105,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D77N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2034,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D77H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4851,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D77Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3092,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D77A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3566,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D77G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2797,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D77V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.395,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D77E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6154,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6154,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F78I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0749,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F78L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4331,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4331,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4331,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F78V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1014,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F78C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1785,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F78S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2541,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F78Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.143,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F79I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1069,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F79L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3823,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3823,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3823,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F79V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1171,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F79C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1489,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F79S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1907,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F79Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1444,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S80R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1563,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1563,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1563,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S80C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0883,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S80G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0889,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S80N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1361,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S80I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1139,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S80T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0932,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q81E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0716,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q81K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0678,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q81R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0688,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q81L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1101,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q81P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0755,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q81H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.141,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.141,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"R82C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1235,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"R82G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.119,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"R82S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2091,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"R82H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0788,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"R82L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1658,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"R82P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2695,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K83Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1989,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K83E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.232,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K83R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0908,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K83I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5538,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"K83T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2725,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K83N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3461,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3461,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V84I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0989,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V84L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1731,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V84F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1198,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V84A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1021,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V84D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1396,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V84G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.111,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I85L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2258,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I85F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.244,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I85V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.188,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I85N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4512,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I85S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4964,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I85T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5285,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I85M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.236,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S86A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1615,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S86P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.166,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S86T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.125,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S86C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.242,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S86F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3423,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S86Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2822,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F87I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.142,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F87L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5547,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5547,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5547,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F87V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1411,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F87C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3315,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F87S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2706,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F87Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1121,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K88Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0829,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K88E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0649,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K88R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0632,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K88M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1697,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K88T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0894,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K88N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1625,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1625,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G89R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3968,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G89C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2961,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G89S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1459,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G89A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1347,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G89D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2005,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G89V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2221,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C90R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6251,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"C90G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4727,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"C90S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6493,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6493,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"C90F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3407,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"C90Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4247,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"C90W\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6025,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"L91I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1087,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L91F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1169,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L91V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1081,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L91R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2744,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L91H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3268,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L91P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4512,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V92I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0877,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V92L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1007,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V92F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0891,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V92A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0683,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V92D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3005,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V92G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0812,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q93E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1699,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q93K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3215,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q93R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.349,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q93L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4567,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q93P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3539,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Q93H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7048,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7048,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"I94L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1322,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1322,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I94V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1524,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I94R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4642,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I94K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5642,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"I94T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5057,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I94M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1268,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F95I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2554,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F95L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7385,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7385,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7385,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F95V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2656,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F95C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3502,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F95S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3707,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F95Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2606,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L96I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1292,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L96F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0735,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L96V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1248,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L96R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4497,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L96H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3421,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L96P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3992,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L97I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1483,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L97F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1785,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L97V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1591,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L97R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3362,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L97H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3407,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L97P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.384,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H98N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3887,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H98D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5624,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H98Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2115,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H98R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3417,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H98L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3185,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H98P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2499,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H98Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6122,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6122,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F99I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0782,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F99L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3735,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3735,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3735,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F99V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0827,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F99C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1153,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F99S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2107,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F99Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1555,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F100I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1431,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F100L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4486,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4486,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4486,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F100V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.146,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F100C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1668,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F100S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1805,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F100Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2593,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G101R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3262,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G101C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.255,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G101S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1457,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G101A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1508,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G101D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4505,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G101V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2889,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G102R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4698,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4698,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G102W\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.519,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G102A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1448,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G102E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.437,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G102V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2216,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S103R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.8434,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.8434,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.8434,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"S103C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0904,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S103G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0778,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S103N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3154,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S103I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2016,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S103T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1012,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E104Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2981,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E104K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2457,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"E104A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4673,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"E104G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4418,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"E104V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5681,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"E104D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5676,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5676,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"M105L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1668,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1668,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M105V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1491,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M105R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5191,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M105K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5096,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M105T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3523,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M105I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4276,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4276,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4276,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V106L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1916,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1916,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V106M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1614,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V106A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1697,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V106E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5229,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V106G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1787,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I107L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0852,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I107F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1797,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I107V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1173,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I107N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3346,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I107S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3536,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I107T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3286,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I107M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1192,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L108I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2716,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L108F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.321,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L108V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3168,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L108R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5554,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L108H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6633,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"L108P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5961,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"I109L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1082,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1082,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I109V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0864,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I109R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4333,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I109K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.558,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I109T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0918,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I109M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1049,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A110P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5689,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"A110S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1311,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A110T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1091,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A110D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.685,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"A110G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1759,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A110V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1568,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M111L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3478,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3478,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M111V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1726,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M111R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.394,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M111K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4731,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M111T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4275,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M111I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7846,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7846,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7846,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"G112R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7232,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"G112C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3337,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G112S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1185,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G112A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0575,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G112D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5673,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"G112V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3831,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F113I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1217,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F113L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6199,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6199,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6199,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"F113V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1444,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F113C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2138,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F113S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3304,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F113Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1503,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"D114N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4763,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D114H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5248,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D114Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4355,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D114A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6844,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"D114G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5582,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"D114V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6579,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"D114E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6991,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6991,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"R115G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4914,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"R115I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6074,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"R115K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3866,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"R115T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.647,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"R115S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7648,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7648,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y116N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4083,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y116D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6501,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y116H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4217,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y116C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2209,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y116F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1396,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y116S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4397,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I117L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1116,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1116,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I117V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.055,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I117R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2225,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I117K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3639,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I117T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0866,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I117M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0829,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A118P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6612,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"A118S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2046,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A118T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2906,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A118E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6888,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"A118G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4041,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"A118V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4047,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I119L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3002,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3002,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I119V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1406,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I119R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.592,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"I119K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7466,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"I119T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5631,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I119M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3206,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C120R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3954,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"C120G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2722,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C120S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3922,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3922,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"C120F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2878,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C120Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3889,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"C120W\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4821,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"K121Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1389,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K121E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2554,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K121R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0682,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K121M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3076,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"K121T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3502,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"K121N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4694,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4694,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P122A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3749,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P122S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5028,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P122T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4071,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P122R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4258,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P122H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4688,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"P122L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.379,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L123I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2465,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L123V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2461,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L123R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2739,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L123Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3327,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L123P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.351,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H124N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2454,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H124D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4191,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H124Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1179,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H124R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0839,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H124L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1492,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H124P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3258,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H124Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2367,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2367,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y125N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3374,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y125D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3932,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y125H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4578,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y125C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3904,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y125F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.283,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"Y125S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4032,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"T126A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0664,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T126P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0646,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T126S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0945,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0945,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T126N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0972,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T126I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1267,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T127A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0661,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T127P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1863,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T127S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1063,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T127R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1097,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T127I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1571,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T127K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1682,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I128L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1294,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I128F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1516,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I128V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0745,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I128N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3321,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I128S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.327,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I128T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2238,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I128M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1275,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M129L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1677,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1677,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M129V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1282,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M129R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2907,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M129K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3618,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M129T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3673,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M129I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4521,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4521,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4521,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"C130R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0776,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C130G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0623,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C130S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.068,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.068,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C130F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1247,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C130Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1137,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C130W\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2076,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G131R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0733,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G131C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1249,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G131S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0825,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G131A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1037,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G131D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.125,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G131V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1134,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N132D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.096,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N132H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0635,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N132Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0827,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N132I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1312,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N132S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0713,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N132T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0997,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"N132K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0935,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0935,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A133P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2509,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A133S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1031,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A133T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0679,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A133E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2304,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A133G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1274,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A133V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0741,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C134R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2837,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C134G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1883,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C134S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2778,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2778,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C134F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1528,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C134Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2434,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"C134W\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3821,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V135I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0755,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V135L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0934,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V135F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0735,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V135A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0762,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V135D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2744,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V135G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1108,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G136R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G136C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1363,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G136S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0997,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G136A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0766,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G136D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3452,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G136V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0878,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I137L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0753,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I137F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0968,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I137V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1089,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I137N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3675,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I137S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3463,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"I137T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2901,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I137M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0835,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M138L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1092,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1092,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M138V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0544,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M138R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4368,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M138K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4816,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"M138T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.102,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"M138I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1973,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1973,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1973,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A139P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2988,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A139S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1352,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A139T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0928,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A139D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4742,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"A139G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1135,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"A139V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0776,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V140I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0779,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V140L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1098,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V140F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0991,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V140A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0744,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V140D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4637,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V140G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.106,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T141A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0543,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T141P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1375,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T141S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0975,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T141R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4629,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"T141I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2115,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"T141K\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5782,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"W142R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6689,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.6689,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"W142G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2485,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"W142L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2809,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"W142S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3007,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"W142C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5141,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5141,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G143R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.587,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.587,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"G143A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0762,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G143E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4347,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G143V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0749,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I144L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1095,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I144F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1059,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I144V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0737,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I144N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2961,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I144S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1625,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I144T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0595,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"I144M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0872,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G145R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5136,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G145C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3094,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G145S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1489,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G145A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1777,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"G145D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5619,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"G145V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3152,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F146I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1001,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F146L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4915,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4915,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4915,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"F146V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1146,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F146C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1726,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F146S\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.292,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"F146Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2708,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L147I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0999,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L147F\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1032,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L147V\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1009,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"L147R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4482,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L147H\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4068,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"L147P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3568,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H148N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4081,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"H148D\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5666,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"H148Y\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2007,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H148R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3248,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H148L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2866,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H148P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.2083,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"H148Q\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.589,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.589,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"S149A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0691,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S149P\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.3495,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S149T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.138,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S149L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1991,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S149W\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.5337,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V150L\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1114,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            },\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1114,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V150M\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.0975,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V150A\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1016,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"V150E\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4205,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"V150G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1391,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S151R\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.7991,\n",
      "                \"Class\": \"likely_pathogenic\"\n",
      "            }\n",
      "        ],\n",
      "        \"S151C\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1095,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S151G\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1192,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S151N\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.4086,\n",
      "                \"Class\": \"ambiguous\"\n",
      "            }\n",
      "        ],\n",
      "        \"S151I\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1709,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ],\n",
      "        \"S151T\": [\n",
      "            {\n",
      "                \"Pathogenicity_Score\": 0.1039,\n",
      "                \"Class\": \"likely_benign\"\n",
      "            }\n",
      "        ]\n",
      "    }\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "# Convert the tree to a JSON string\n",
    "json_tree = json.dumps(tree, indent=4)\n",
    "print(json_tree)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "96569321-527c-4cf6-9a9e-5714f708d5a0",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "hacklowe",
   "language": "python",
   "name": "hacklowe"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
